core/tests/protractor/learnerDashboard.js

```diff
@@ -129,17 +126,17 @@ describe('Learner dashboard functionality', function() {
     // 'In Progress' section.
     player.submitAnswer('Continue', null);
     browser.ignoreSynchronization = true;
-    browser.get(general.LEARNER_DASHBOARD_URL);
+    learnerDashboardPage.get();
     general.acceptAlert();
     browser.waitForAngular();
     general.waitForSystem();
     browser.ignoreSynchronization = false;
-    element(by.css('.protractor-test-incomplete-collection-section')).click();
+    learnerDashboardPage.incompleteCollectionSection();
```

Comment:
Ditto: method names should start with verbs and accurately describe what they do semantically.

Reply:
Done, I have renamed it.